sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -56,4 +51,8 @@ private static void createsEnvelopeDirPath(SentryOptions options, Context contex
     }
     options.setCacheDirPath(envelopesDir.getAbsolutePath());
   }
+
+  private static boolean isNdkAvailable() {
```

Comment:
Are we settling on the fact that NDK support will not be available on API < 21?

Reply:
Yeah, tried to build the NDK feature branch on API 14 and got an error on that method :(

Reply:
we might have a workaround, parsing `/proc/PID/maps` by hand and getting what we need.
but still thinking if it's a good idea.